01-data-structures/03-linked-lists/linked_list.rb

```diff
@@ -6,25 +6,118 @@ class LinkedList
 
   # This method creates a new `Node` using `data`, and inserts it at the end of the list.
```

Comment:
This was more of a comment for you on why I added these methods. (due to the Benchmark requirement)